Dockerfile

```diff
@@ -1,17 +1,13 @@
-FROM openjdk:8-jdk AS compile
+FROM debian:stretch-slim AS compile
```

Comment:
Since we're downloading Java through SDKMAN anyway, I replaced this with smaller base image:
![2021-03-09_16-47](https://user-images.githubusercontent.com/5559568/110559084-73eba580-80f8-11eb-9389-025e2c539a2d.png)